aws/provider.go

```diff
@@ -16,13 +16,14 @@ import (
 	tfaws "github.com/terraform-providers/terraform-provider-aws/aws"
 )
 
-// skippableErrors is a list of errors
+// skippableCodes is a list of errors
```

Comment:
Hmm, should it be `a list of codes` since you changed name from `...Errors` to `...Codes` ?